buildlib/dockers/docker-compose-aarch64.yml

```diff
@@ -0,0 +1,55 @@
+version: "3.4"
+
+# How to:
+# export ARCH=$(uname -m)
+# docker compose -f docker-compose-${ARCH}.yml build
+
+services:
```

Comment:
docker-compose-aarch64.yml docker-compose-x86_64.yml still seem to have a lot of "code duplication", can we have common yaml file for them?
how do they differ from one another?

Reply:
1. MOFED and CUDA support matrixes differ between CPU architectures.
2. Not a single image spec is shared between the two yml files.
3. Docker compose does not support a conditional statement.
With the existing manual process of building images in place, I see no other choice but to keep the two yml files separated.

Reply:
Your wish was granted. Please, review.

Reply:
this file is docker-compose-**aarch64**.yml
Maybe it should just define `ARCH: aarch64` ?
same for x86_64

Reply:
Right, those are leftovers of attempts to select platforms dynamically,